lib/index.js

```diff
@@ -14,33 +19,34 @@ const GRAMMAR_SCOPES = [
   'text.html.ruby'
 ];
 
-export const config = {
-  executablePath: {
-    title: 'Executable Path',
-    description: 'HTMLHint Node Script Path',
-    type: 'string',
-    default: path.join(__dirname, '..', 'node_modules', 'htmlhint', 'bin', 'htmlhint')
-  }
-};
-
-let executablePath = '';
+export const config = {};
 
 export function activate() {
   require('atom-package-deps').install('linter-htmlhint');
+}
 
-  executablePath = atom.config.get('linter-htmlhint.executablePath');
-
-  atom.config.observe('linter-htmlhint.executablePath', newValue => {
-    executablePath = newValue;
-  });
+function getConfig(filePath) {
+  return findAsync(path.dirname(filePath), '.htmlhintrc')
+    .then(configPath => {
```

Comment:
I was merely suggesting `cosmiconfig` as an option for not dealing with loading and parsing the file. Your second point of concern there is what I meant by "extra functionality compared to `htmlhint` itself.

I'm perfectly fine with how this is implemented now, so let's just go with that :wink: